aries-site/src/components/content/ThemeModeToggle.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Button } from 'grommet';
-import { IconMoon, IconSun } from '../icons';
+import { Moon as IconMoon, Sun as IconSun } from 'grommet-icons';
```

Comment:
Do we want to leave with no alias and just change the lines below to reference `Moon` and `Sun`?

Reply:
Sure, done.